dotnet/src/SemanticKernel/AI/AIException.cs

```diff
@@ -104,15 +121,13 @@ private AIException()
         // Not allowed, error code is required
     }
 
-    private AIException(string message) : base(message)
+    protected AIException(string message) : base(message)
```

Comment:
I would not change the visibility. The intent is to make error codes mandatory, ie always provide specific errors.

Reply:
Not sure why i did this. Changing.

Reply:
Ah, my IDE did this. WIll fix.